usersettings.py

```diff
@@ -2,9 +2,15 @@
 """
 Provide interface for persistent portable editable user settings
 """
-import os
-import ConfigParser
 import ast
+import os
+import sys
+
+PY3 = sys.version_info[0] == 3
+if PY3:
+    import configparser
+else:
+    import ConfigParser as configparser
```

Comment:
Actually, it should probably be changed to something like this:

    try:
        import configparser
    except ImportError:
        import ConfigParser as configparser